### PR TITLE
Fix minor errors in docs

### PR DESCRIPTION
**Pull request by rhooper**
Fixes docs:
- Serpentine => serpentine
- tiles_rows =>  tile_rows

### CODE REVIEW

File: adafruit_matrixportal/matrixportal.py [hunk 1]
```diff
@@ -63,10 +63,10 @@ class MatrixPortal(PortalBase):
     :param debug: Turn on debug print outs. Defaults to False.
     :param int width: The total width of the display(s) in Pixels. Defaults to 64.
     :param int height: The total height of the display(s) in Pixels. Defaults to 32.
-    :param bool Serpentine: Used when panels are arranged in a serpentine pattern rather
+    :param bool derpentine: Used when panels are arranged in a serpentine pattern rather
```
Comment (dhalbert) [inline adafruit_matrixportal/matrixportal.py]: :smiley: 
```suggestion
    :param bool serpentine: Used when panels are arranged in a serpentine pattern rather
```

**Review by dhalbert — CHANGES_REQUESTED** [review]

**Comment by dhalbert** [comment]
Something is busted about the build -- not sure what the problem is, since your change was so innocuous.

**Comment by dhalbert** [comment]
@makermelissa Do you understand why the build is failing?

**Comment by FoamyGuy** [comment]
@dhalbert and @makermelissa 

It appears there has been a recent new major release for [pillow](https://github.com/python-pillow/Pillow/releases)

And amongst the release notes it lists this:

> Remove deprecated ImageFont.getsize and related functions for Pillow 10.0.0 https://github.com/python-pillow/Pillow/pull/7080 [@radarhere]

In Blinka_Displayio it's attempting to use the `getsize()` function here: https://github.com/adafruit/Adafruit_Blinka_Displayio/blob/61102c93bfb90fbccc4bc21122444531ebe2dd84/fontio.py#L78

I'm not entirely sure why it manifested itself in this particular way on this PR, but I think the options to resolve it for now are:

- pin `pillow` to a lower release version number
- Change fontio.py in Blinka_Displayio to use a different non-deprecated API, hopefully the functionality has just moved elsewhere and not been removed entirely.

**Comment by radarhere** [comment]
Hi.

> Change fontio.py in Blinka_Displayio to use a different non-deprecated API, hopefully the functionality has just moved elsewhere and not been removed entirely.

You can read at https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#font-size-and-offset-methods that `ImageFont.getsize()` has been replaced by `ImageFont.getbbox()`

Edit: I now see this has been done at https://github.com/adafruit/Adafruit_Blinka_Displayio/pull/108

**Comment by FoamyGuy** [comment]
Thank you. Sorry to ping you, I didn't think to pull the "@" out of the line I copied from the release notes.

**Comment by makermelissa** [comment]
> I'm not entirely sure why it manifested itself in this particular way on this PR, but I think the options to resolve it for now are:
> 
> * pin `pillow` to a lower release version number
> * Change fontio.py in Blinka_Displayio to use a different non-deprecated API, hopefully the functionality has just moved elsewhere and not been removed entirely.

I'm fairly sure it installs Blinka Displayio unless you add **displayio** to `autodoc_mock_imports` in https://github.com/adafruit/Adafruit_CircuitPython_MatrixPortal/blob/main/docs/conf.py, so that's another possible fix. However, I think fixing it it in Blinka_Displayio would be the preferred fix.

**Comment by makermelissa** [comment]
@dhalbert the build is now passing if you want to approve unless you had some other requested changes.

**Review by FoamyGuy — APPROVED** [review]
This looks good to me.

we can always use a new PR if there are any further changes that come up.

**Pull request merged**